providers/yandex/tests/unit/yandex/hooks/test_dataproc.py

```diff
@@ -21,7 +21,7 @@
 
 import pytest
 
-yandexlcloud = pytest.importorskip("yandexcloud")
+yandexcloud = pytest.importorskip("yandexcloud")
```

Comment:
Good that you fixed the typo... but the variable is not used below? Is the variable needed then?

Reply:
Not needed. removed.